src/SFA.DAS.ApplyService.Web/Views/Users/CreateAccount.cshtml

```diff
@@ -61,8 +83,17 @@
                     {
                       <span class="govuk-error-message" asp-validation-for="Email"></span>
                     }
-                    <input class="govuk-input" id="Email" name="Email" type="text" asp-for="Email"
-                        aria-describedby="email-hint" sfa-validation-for="FamilyName" sfa-validationerror-class="govuk-input--error" />
+                    <input
+                        class="govuk-input"
+                        id="Email"
+                        name="Email"
+                        type="text"
+                        asp-for="Email"
+                        aria-describedby="email-hint"
+                        sfa-validation-for="FamilyName"
+                        sfa-validationerror-class="govuk-input--error"
+                        maxlength="320"
```

Comment:
There doesn't seem to be anything specific in the ticket that suggests max length would be changed - or what the correct value should be (I'm guessing that this is the Email field in [Apply].[Contacts] - which is 256 or could eventually end up there so would be best to be the same max length)